src/utils/SharedFunctions.js

```diff
@@ -320,3 +319,7 @@ export const uniqueYears = (allYears) => {
       return { active: currentYear === year ? true : false, year: year };
   });
 };
+
+export const updateProducerInformation = async (data, getTokenSilently) => {
```

Comment:
adding another wrapper here seems redundant. please import and call `callAzureFunction()` in whichever file you use it in instead of making another wrapper here and importing that